lib/chef/knife/bootstrap.rb

```diff
@@ -146,6 +147,12 @@ class Bootstrap < Knife
         description: "The version of Chef to install",
         proc: lambda { |v| Chef::Config[:knife][:bootstrap_version] = v }
 
+      option :channel,
+        long: "--channel CHANNEL",
+        description: "Install from the given channel. Valid values are 'current' and 'stable'. Default is 'stable'",
+        default: "stable",
+        in: %w{stable current}
```

Comment:
Can we allow `unstable` here too? There are testing scenarios where we would want the ease of grabbing a build from that channel and it is accessible.